pixart.js

```diff
@@ -1 +1,11 @@
+var brush = document.querySelector(".brush");
+var square = document.querySelector(".square");
+var form = document.querySelector(".form");
+var input = document.querySelector(".set-color");
+var controls = document.querySelector(".controls");
 
+input.addEventListener("click", changeColor);
+
+function color () {
+input.innerHTML = brush;
+}
```

Comment:
you're off to a good start! Let me know if yo'd like to work through this one together sometime this week